remove hardcoded paths from scripts
authors: Denis Kristak (INUTS) + Kenneth Hoste (HPC-UGent)
diff -ru r-inla-Version_24.01.18.orig/rinla/inst/bin/linux/64bit/fmesher.run r-inla-Version_24.01.18/rinla/inst/bin/linux/64bit/fmesher.run
--- r-inla-Version_24.01.18.orig/rinla/inst/bin/linux/64bit/fmesher.run	2024-01-18 20:34:05.000000000 +0100
+++ r-inla-Version_24.01.18/rinla/inst/bin/linux/64bit/fmesher.run	2024-01-19 15:51:25.789786893 +0100
@@ -4,17 +4,6 @@
 prog=${tmp%%.run}
 DIR=$(dirname "$cmd")
 
-D=""
-for d in {,/usr}/lib64 /usr/lib64/R/lib {,/usr}/lib/x86_64-linux-gnu {,/usr}/lib; do
-    if [ -d "$d" ]; then
-	if [ -z "$D" ]; then
-	    D="$d"
-	else
-	    D="$D:$d"
-	fi
-    fi
-done
-export LD_LIBRARY_PATH="$DIR/first:$D:$DIR:$LD_LIBRARY_PATH"
 
 if [ "${INLA_DEBUG}XX" != "XX" ]; then
     echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH"
diff -ru r-inla-Version_24.01.18.orig/rinla/inst/bin/linux/64bit/inla.run r-inla-Version_24.01.18/rinla/inst/bin/linux/64bit/inla.run
--- r-inla-Version_24.01.18.orig/rinla/inst/bin/linux/64bit/inla.run	2024-01-18 20:34:05.000000000 +0100
+++ r-inla-Version_24.01.18/rinla/inst/bin/linux/64bit/inla.run	2024-01-19 15:56:17.584566196 +0100
@@ -13,17 +13,6 @@
     fi
 fi
 
-D=""
-for d in {,/usr}/lib64 /usr/lib64/R/lib {,/usr}/lib/x86_64-linux-gnu {,/usr}/lib; do
-    if [ -d "$d" ]; then
-	if [ -z "$D" ]; then
-	    D="$d"
-	else
-	    D="$D:$d"
-	fi
-    fi
-done
-
 for f in $DIR/first/lib*.so*; do
     case "$f" in
 	$DIR/first/libjemalloc.so*)
